### PR TITLE
[Bug Fix] Fix log messages when players join channel

### PR DESCRIPTION
**Pull request by Valorith**
Removes trailing commas/spaces from UCS log messages when no saved channels exist during load.

### CODE REVIEW

File: ucs/clientlist.cpp [hunk 1]
```diff
@@ -793,7 +793,10 @@ void Clientlist::ProcessOPMailCommand(Client *c, std::string command_string, boo
 	case CommandJoin:
 		if (!command_directed) {
 			//Append saved channels to params
-			parameters = parameters + ", " + database.CurrentPlayerChannels(c->GetName());
+			auto const savedChannels = database.CurrentPlayerChannels(c->GetName());
+			if (savedChannels.length() > 0) {
+				parameters = parameters + ", " + savedChannels;
```
Comment (Kinglykrab) [inline ucs/clientlist.cpp:798]: `parameters += fmt::format(“, {}”, savedChannels);`

Comment (Valorith) [reply ucs/clientlist.cpp]: Updated

**Review by Kinglykrab — APPROVED** [review]

**Comment by Kinglykrab** [comment]
Looks good, assuming this has been tested.

File: ucs/clientlist.cpp [hunk 2]
```diff
@@ -793,7 +793,10 @@ void Clientlist::ProcessOPMailCommand(Client *c, std::string command_string, boo
 	case CommandJoin:
 		if (!command_directed) {
 			//Append saved channels to params
-			parameters = parameters + ", " + database.CurrentPlayerChannels(c->GetName());
+			auto const savedChannels = database.CurrentPlayerChannels(c->GetName());
+			if (savedChannels.length() > 0) {
+				parameters += fmt::format(", {}", savedChannels);
+			}
```
Comment (Kinglykrab) [inline ucs/clientlist.cpp:799]: `const auto`, would name it `saved_channels` as well.

Comment (Kinglykrab) [reply ucs/clientlist.cpp:799]: Not that `auto const` and `const auto` aren't the same, it's just easier to read and fits with other places that do this. Variables are generally snake-cased.

**Comment by Valorith** [comment]
Please do not merge yet. Working through some changes in this code suggest by @Akkadius

**Comment by Valorith** [comment]
This is ready for a fresh review, when able. Thanks

File: ucs/clientlist.cpp [hunk 3]
```diff
@@ -793,7 +793,10 @@ void Clientlist::ProcessOPMailCommand(Client *c, std::string command_string, boo
 	case CommandJoin:
 		if (!command_directed) {
 			//Append saved channels to params
-			parameters = parameters + ", " + database.CurrentPlayerChannels(c->GetName());
+			const auto saved_channels = database.CurrentPlayerChannels(c->GetName());
+			if (!saved_channels.empty()) {
+				parameters += fmt::format(", {}", Strings::Join(saved_channels, ", "));
```
Comment (Akkadius) [inline ucs/clientlist.cpp:798]: Did you test this - don't think this will do what you think it will

Comment (Valorith) [reply ucs/clientlist.cpp:798]: Seems to work fine in my testing

**Comment by Valorith** [comment]
No more trailing commas in UCS Channel log statements
image

**Pull request merged**